benchmark/daisy/daisy.jl

```diff
@@ -0,0 +1,70 @@
+# ======================
+# Initialization
+# ======================
+
+# The benchmarks in this file are taken from
+# [Project Daisy](https://github.com/malyzajko/daisy/blob/master/testcases/).
+SUITE["Daisy"] = BenchmarkGroup()
+RESULTS = SUITE["Daisy"]
+
+# Dictionaries to store data and reference values
+DAISY_1D = Dict{String, Any}()
+DAISY_ND = Dict{String, Any}()
+
+include("setup.jl")
+include("univariate.jl")
+include("multivariate.jl")
+
+# ===============================
+# Run one-dimensional benchmarks
+# ===============================
+for name in keys(DAISY_1D)
+    RESULTS[name] = BenchmarkGroup()
+    RELPREC[name] = Dict()
+    dom = DAISY_1D[name]["dom"]
+    ref = DAISY_1D[name]["ref"]
+    for m in [2, 5, 10]
```

Comment:
I recommend using variable names that somehow represent the quantity. In this case, instead of using `m` I would suggest to use `ord` or `order`, which reflect what those integer values are. (I admit that this is described below, but still think that my suggestion yields clarity.)